lib/migrations/add_cross_assignments_table.rb

```diff
@@ -0,0 +1,13 @@
+class AddCrossAssignmentsTable < ActiveRecord::Migration
+  def change
+    create_table :cross_assignments do |t|
+      t.integer :cross_parent_id, null: false
```

Comment:
I'd suggest making the column names here be the same as those in Assignment to make it easier to DRY the code.